dpnp/backend/extensions/elementwise_functions/elementwise_functions.hpp

```diff
@@ -859,6 +859,279 @@ py::object py_binary_ufunc_result_type(const py::dtype &input1_dtype,
     }
 }
 
+/*! @brief Template implementing Python API for querying of type support by
+ *         binary elementwise functions */
+template <typename output_typesT,
+          typename contig_dispatchT,
+          typename strided_dispatchT>
+std::pair<sycl::event, sycl::event>
+    py_binary_two_outputs_ufunc(const dpctl::tensor::usm_ndarray &src1,
+                                const dpctl::tensor::usm_ndarray &src2,
+                                const dpctl::tensor::usm_ndarray &dst1,
+                                const dpctl::tensor::usm_ndarray &dst2,
+                                sycl::queue &exec_q,
+                                const std::vector<sycl::event> depends,
```

Comment:
depends should be `const std::vector<sycl::event> &` as it is done in dpctl `py_unary_ufunc`